src/index.js

```diff
@@ -1,106 +1,104 @@
+/* eslint-disable import/no-cycle */
+
 import './style.css';
 import completedStatus from './complete.js';
+import addTaskList from './add-to-list.js';
+import removeTask from './remove.js';
 // import { templateSettings } from 'lodash';
 
-const form = document.getElementById('my-form');
 const allTasks = document.getElementById('all-tasks');
 const insert = document.getElementById('enter-task');
-const addTask = document.getElementById('task');
+const removeAll = document.getElementById('all-completed');
 
-let taskList = [
-  {
-    description: 'Wash clothes',
-    completed: false,
-    id: 0,
-  },
-  {
-    description: 'Spend Time With Family',
-    completed: false,
-    id: 1,
-  },
-  {
-    description: 'Read Books',
-    completed: false,
-    id: 2,
-  },
-];
+let taskList = [];
 
 const getData = () => {
   if (localStorage.getItem('taskList') !== null) {
-    taskList = JSON.parse(localStorage.getItem('tasklist'));
+    taskList = JSON.parse(localStorage.getItem('taskList'));
   }
 };
 
 const displayTasks = () => {
```

Comment:
- [OPTIONAL ] In order not to have too long functions, consider extracting event listeners out of the functions. This is necessary for code readability. Concise === readability.

Reply:
Hi Lameck, thanks for sharing your feedback. I will implement these optional changes over the weekend.